app/views/users/user_new.erb

```diff
@@ -0,0 +1,20 @@
+
+
+
+
+<h1>Act Now time is running out!</h1>
+<p></p>
+
+<form action = "/user/user_create">
```

Comment:
For this form to submit to the `MyApp.post "/login/create" do` action, the form's `method` must be `post`.